portfolio/src/main/webapp/script.js

```diff
@@ -82,6 +83,49 @@ function showCaption(element) {
   caption.focus();
 }
 
+/*
+ * Fetch the authentication status of the user from the server
+ */
+ function authStatus() {
+   let loginStatus = false;
+
+   fetch('/login')
+  .then(response => response.json())
+  .then((login) => {
+    // login is a JSON object
```

Comment:
TECHNICALLY all objects in js are JSON objects, as it's just the JavaScript Object Notation :)

Reply:
oh that makes sense, good to know!